### DISCUSSION TITLE
Disable auotfill saving

### DISCUSSION THREAD

**Discussion by trymeouteh** [Kunzisoft/KeePassDX#1737] [Ideas]
Have a setting in the autofill settings to disable the users from adding or editing entries in a database when auotfilling a form.

Userful for users who want to create entries in their databases manually and not have this automated when signing up to a service.

**Comment by J-Jamet** [comment]
Issue : https://github.com/Kunzisoft/KeePassDX/issues/1747

**Reply by J-Jamet** [reply]
Simply disable `Settings - Autofill settings - Save search info` and `Settings - Autofill settings - Manual selection`
